microsoft-edge/webview2/release-notes.md

```diff
@@ -33,29 +33,29 @@ To use a prerelease SDK along with a Microsoft Edge preview channel, navigate to
 
 
 <!-- ====================================================================== -->
-<!-- ## 1.0.1053-prerelease
+## 1.0.1053-prerelease
 
 Release Date: October 27, 2021
```

Comment:
@jasonstephen15 - https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1053-prerelease 
```suggestion
Release Date: October 29, 2021
```
gives message "The specified version 1.0.1053-prerelease was not found."